### PR TITLE
Fix valgrind uninitialized errors in EB coloring

### PR DESCRIPTION
**Pull request by brian-kelley**
While doing #1528 I saw some other conditional branch on uninitialized errors in EB distance-1 coloring. tentative_color_ban was not initialized at construction but should have been.

### CODE REVIEW

**Review by lucbv — APPROVED** [review]
Looks good!

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection Is Not Necessary for this Pull Request.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC930_Light_Tpls_GCC930</h3>
<ul>
  <li>Build Num: 37</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CUDA11_CUDA11_LayoutRight</h3>
<ul>
  <li>Build Num: 46</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720</h3>
<ul>
  <li>Build Num: 99</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720_Light_LayoutRight</h3>
<ul>
  <li>Build Num: 97</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720</h3>
<ul>
  <li>Build Num: 97</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL19</h3>
<ul>
  <li>Build Num: 48</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CLANG1001</h3>
<ul>
  <li>Build Num: 97</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: KOKKOSKERNELS (brian-kelley/kokkos-kernels)</dt>
  <ul>
    <li>Branch: FixColoringUninitialized</li>
    <li>SHA: 62fdbabd1e17fb1719e76803995874f803196d6f</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: brian-kelley
</details>

**Comment by brian-kelley** [comment]
Note to self: wait until #1504 is merged before merging this (will have to rebase)

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Jenkins Testing: all Jobs PASSED

<details>
<summary>
  <b>Pull Request Auto Testing has PASSED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC930_Light_Tpls_GCC930</h3>
<ul>
  <li>Build Num: 37</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CUDA11_CUDA11_LayoutRight</h3>
<ul>
  <li>Build Num: 46</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720</h3>
<ul>
  <li>Build Num: 99</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720_Light_LayoutRight</h3>
<ul>
  <li>Build Num: 97</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720</h3>
<ul>
  <li>Build Num: 97</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL19</h3>
<ul>
  <li>Build Num: 48</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CLANG1001</h3>
<ul>
  <li>Build Num: 97</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>FixColoringUninitialized</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/brian-kelley/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>62fdbabd1e17fb1719e76803995874f803196d6f</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>c7fd5ad259908ed82153b753ef4f00e7f0ecf452</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1530</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>
</details>

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Merge Inspection' - SUCCESS: The last commit to this Pull Request has been INSPECTED AND APPROVED by [ lucbv ]!

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Pull Request MUST BE MERGED MANUALLY BY Project Team - This Repo does not support Automerge

**Comment by ndellingwood** [comment]
Cherry-picked to #1640 (3.7.01 fixup) sha [f5d1ee5](https://github.com/kokkos/kokkos-kernels/pull/1640/commits/f5d1ee50407ad4e1beea4849e892f6ec81695baf)

**Pull request merged**
